crates/core/tedge_mapper/src/core/mapper.rs

```diff
@@ -103,21 +103,18 @@ impl Mapper {
         for init_message in init_messages.into_iter() {
             let _ = self.output.send(init_message).await;
         }
-
```

Comment:
Minor. Why did you update this file adding/removing blank lines?

Reply:
I added some debug statements while adding new tests, and forgot to restore it to the previous state after removing those debug statements.